README.md

```diff
@@ -1,7 +1,5 @@
 ## Specification and Bindings for Swift Binary Protocol
 
-[![Build status][1]][2]
```

Comment:
So we can probably add this back later, but it would be for all languages, instead of just one global "CI" badge?